z64.h

```diff
@@ -0,0 +1,306 @@
+#define z64_frame_input_call_func_addr			0x800A0BA0
+#define z64_frame_input_call_call_addr			0x800A16AC
+#define z64_SetTextRGBA_addr					0x800CBE58
+#define z64_SetTextXY_addr						0x800CBEC8
+#define z64_SetTextString_addr					0x800CC588
+#define z64_active_timer_addr					0x8011A5DC
+#define z64_max_hearts_addr						0x8011A5FE
+#define z64_hearts_addr							0x8011A600
+#define z64_update_magic_func_addr				0x8011A602
+#define z64_magic_addr							0x8011A603
+#define z64_rupees_addr							0x8011A604
+#define z64_has_magic_addr						0x8011A60A
+#define z64_has_dbl_magic_addr					0x8011A60C
+#define z64_equipment_3_addr					0x8011A60E
+#define z64_b_btn_itm_id_addr					0x8011A638
+#define z64_cl_btn_itm_id_addr					0x8011A639
+#define z64_cd_btn_itm_id_addr					0x8011A63A
+#define z64_cr_btn_itm_id_addr					0x8011A63B
+#define z64_equipment_ptr_addr					0x8011A640
+#define z64_deku_stick_addr						0x8011A644
+#define z64_deku_nut_addr						0x8011A645
+#define z64_bomb_addr							0x8011A646
+#define z64_bow_addr							0x8011A647
+#define z64_arrow_fire_addr						0x8011A648
+#define z64_dins_fire_addr						0x8011A649
+#define z64_slingshot_addr						0x8011A64A
+#define z64_ocarina_addr						0x8011A64B
+#define z64_bombchu_addr						0x8011A64C
+#define z64_hookshot_addr						0x8011A64D
+#define z64_arrow_ice_addr						0x8011A64E
+#define z64_farores_wind_addr					0x8011A64F
+#define z64_boomerang_addr						0x8011A650
+#define z64_lens_of_truth_addr					0x8011A651
+#define z64_magic_bean_addr						0x8011A652
+#define z64_megaton_hammer_addr					0x8011A653
+#define z64_arrow_light_addr					0x8011A654
+#define z64_nayrus_love_addr					0x8011A655
+#define z64_bottle_1_opt_addr					0x8011A656
+#define z64_bottle_2_opt_addr					0x8011A657
+#define z64_bottle_3_opt_addr					0x8011A658
+#define z64_bottle_4_opt_addr					0x8011A659
+#define z64_trade_itm_adult_addr				0x8011A65A
+#define z64_trade_itm_child_addr				0x8011A65B
+#define z64_qty_deku_stick_addr					0x8011A65C
+#define z64_qty_deku_nut_addr					0x8011A65D
+#define z64_qty_bomb_addr						0x8011A65E
+#define z64_qty_arrow_addr						0x8011A65F
+#define z64_qty_deku_seed_addr					0x8011A662
+#define z64_qty_bombchu_addr					0x8011A664
+#define z64_qty_magic_bean_addr					0x8011A66A
+#define z64_equipment_1_addr					0x8011A66C
+#define z64_equipment_2_addr					0x8011A66D
+#define z64_upgrades_addr						0x8011A670
+#define z64_quest_itm_1_addr					0x8011A674
+#define z64_quest_itm_2_addr					0x8011A675
+#define z64_quest_itm_3_addr					0x8011A676
+#define z64_quest_itm_4_addr					0x8011A677
+#define z64_dungeon_itm_addr					0x8011A678
+#define z64_dungeon_key_addr					0x8011A68C
+#define z64_defense_hearts_addr					0x8011A69F
+#define z64_gs_tokens_addr						0x8011A6A1
+#define z64_reset_gs_addr						0x8011B46C
+#define z64_flag_swordless_addr					0x8011B503
+#define z64_flag_interface_addr					0x8011B92F
+#define z64_flag_nayrus_addr					0x8011B998
+#define z64_countdown_tmr_addr					0x8011B9A2
```

Comment:
I intended to add this to the current set of features. It freezes the currently active timer which should allow for practice of void warp or surviving various conditions without the correct equipment, e.g: underwater.